source/reference/collection-level-filtering.txt

```diff
@@ -0,0 +1,258 @@
+      If you don't specify a filter, ``mongosync`` does a full cluster
+      sync.
+
+      The filter syntax is: 
+
+      .. include:: /includes/api/facts/includeNamespaces-syntax.rst
+
+      Create an entry in the ``includeNamespaces`` array for each
+      database that you identified in step 1. Use the ``"database"``
+      field to specify the database name.
+      
+      If you want to filter on collections within a database, add those
+      collections to the list in the ``"collections"`` field for that
+      database entry.
+
+   .. step:: Use the Filter.
+
+      To use the filter, attach the filter json when you make the
+      :ref:`/start <c2c-api-start>` API call to begin syncing.
+
+      .. literalinclude:: /includes/api/requests/start-filtered.sh
+         :language: shell
+
+For an example configuration, see: :ref:`c2c-example-start-with-filter`.
+
+.. _c2c-change-filter:
+
+Replace an Existing Filter
+--------------------------
+
+You cannot update an existing filter. You must stop the ongoing sync
+process, prepare the destination cluster, and restart ``mongosync`` with
+a new filter. 
+
+When ``mongosync`` ran your original filter, it created databases with
+your data ("user databases") and the
+``mongosync_reserved_for_internal_use`` system database on the
+destination cluster. You must remove those databases before restarting
+``mongosync`` with your new filter.
+
+Follow these steps to prepare the destination cluster for a new filter.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Remove ``mongosync_reserved_for_internal_use``.
+
+      a. Stop the ``mongosync`` process. 
+      
+      #. Connect to the destination cluster with :binary:`mongosh`. If
+         the destination is a sharded cluster, connect to the
+         :binary:`mongos` instance. If the destination is a replica set,
+         connect to the primary :binary:`mongod` instance.
+
+      #. Drop the ``mongosync_reserved_for_internal_use`` system
+         database.
+
+         .. code-block:: javascript
+          
+            use mongosync_reserved_for_internal_use
+            db.dropDatabase()
+
+   .. step:: Remove user databases.
+
+      a. List the databases in the cluster
+
+         .. code-block:: javascript
+          
+            show databases
+
+      #. Remove user databases. The ``admin``, ``local``, and ``config``
+         databases are system databases. You should not edit these
+         system databases without instructions from MongoDB support. 
+         
+         If the ``show databases`` command lists any user databases on
+         the destination cluster, you must remove them.
+
+         Repeat this step for each user database list:
+
+         .. code-block:: javascript
+          
+            use <user database name>
+            db.dropDatabase()
+
+         Note: After the first :method:`db.dropDatabase()` operation
+         completes, you may need to run it a second time to remove the
+         database.
```

Comment:
The admonition is in the docs for dropDatabase. 

Initially I had the command written twice because when I first started experimenting with filtering I had to drop the db twice several times. Later on it seemed to work ok the first. Rohan noted that dropping consistently worked the first time for him. Given that, I decided on a note instead of the doubled command.